apps/domain/test/jobs/new_release_job_test.exs

```diff
@@ -96,12 +110,26 @@ defmodule Domain.Jobs.NewReleaseJob.Test do
 
     assert job1 == %NewReleaseJob{
              new_tag: tag1,
-             repo: %Repo{name: "elixir", owner: "elixir-lang", tags: []}
+             repo: %Repo{
+               id: 1,
+               url: "https://github.com/elixir-lang/elixir",
+               name: "elixir",
+               owner: "elixir-lang",
+               tags: [],
```

Comment:
Worth considering, later when we build on every commit - there could be thousands of commits per repo